traitsui/examples/demo/Standard_Editors/TreeEditor_demo.py

```diff
@@ -7,9 +7,9 @@
 In this case, the tree has the following hierarchy:
 
 - Partner
- - Company
-  - Department
-   - Employee
+    - Company
+        - Department
+            - Employee
```

Comment:
without proper indentation, this was raising warnings, which is fixed by adding tabs for indentation.